### ISSUE TITLE
Pretraining resume error on XLSR Model wav2vec2

### ISSUE THREAD

**Issue by harveenchadha** [pytorch/fairseq#3258]
## 🐛 Bug

When I try to resume the training on XLSR it does get resumes from 19th epoch onwards as it is shown below:

<img width="1785" alt="Screenshot 2021-02-20 at 2 41 14 PM" src="https://user-images.githubusercontent.com/30959215/108590454-b20f6800-7389-11eb-991d-b75b71cc9b8c.png">

But when after the epoch I try to save the model, I get this error:

<img width="1792" alt="Screenshot 2021-02-20 at 2 42 14 PM" src="https://user-images.githubusercontent.com/30959215/108590482-d4a18100-7389-11eb-8786-962f122a5832.png">

I have seen this is only with the case of XLSR.

If I set reset-dataloader to True in the config, the training gets restarted from epoch 1 instead of 19, the loss and other things remain the same, however the learning rate gets impacted and starts from a very high value.

**Comment by olafthiele** [comment]
Did you change any of the parameters mentioned in the error msg? Like nr of GPUs or the train/valid material?
